docs/run/sessions.mdx

```diff
@@ -86,6 +88,8 @@ session.close()
 ## Next steps
 
 <Admonition type="tip" title="Recommendations">
```

Comment:
A comment on the first sentence under `Usage patterns`. Right now it says `Sessions run iteratively.`, which is an odd expression. Can we just change it to something like "Sessions are especially useful for algorithms that require classical post-processing, ..."